### DISCUSSION TITLE
Connecting to remote database

### DISCUSSION THREAD

**Discussion by Xtonx** [xenocrat/chyrp-lite#171] [Q&A] [answered]
Hi Al,
I've got a centralized db server going already and would prefer to use that over another DB on the local Chyrp VM.

Is it possible to connect Chyrp-Lite to a MySQL database running on a different machine? If so I would greatly appreciate any advice or instructions on how to do so. The only option I'm seeing on the install.php page is to setup a local SQLite db. Forgive me if this is somewhere in the documentation, I wasn't able to find it on the Wiki or going through the existing issues list.

**Comment by xenocrat** [comment, marked as answer]
Hello there,

Chyrp Lite supports SQLite, MySQL, and PostreSQL, but you must have your PHP installtion configured with [PDO driver support](https://www.php.net/manual/en/pdo.installation.php) in order to use them. Once done, the installer should look like [this](https://chyrplite.net/wiki/images/install.png).

**Reply by Xtonx** [reply]
Perfect, thank you very much! Looks like the PDO_MySQL extension wasn't in my php.ini file, works great now.
